cmd/notation/verify.go

```diff
@@ -1,39 +1,34 @@
 package main
 
 import (
-	"context"
 	"errors"
-	"fmt"
 	"os"
 
-	"github.com/notaryproject/notation-core-go/signature/jws"
-	"github.com/notaryproject/notation-go"
-	"github.com/notaryproject/notation-go/dir"
-	"github.com/notaryproject/notation-go/signature"
+	"github.com/notaryproject/notation-go/registry"
+	"github.com/notaryproject/notation-go/verification"
 	"github.com/notaryproject/notation/internal/cmd"
-	"github.com/notaryproject/notation/internal/slices"
-	"github.com/notaryproject/notation/pkg/cache"
-	"github.com/notaryproject/notation/pkg/configutil"
-	"github.com/opencontainers/go-digest"
+	"github.com/notaryproject/notation/internal/ioutil"
+
+	orasregistry "oras.land/oras-go/v2/registry"
+
 	"github.com/spf13/cobra"
 )
 
 type verifyOpts struct {
-	RemoteFlagOpts
-	signatures []string
-	certs      []string
-	certFiles  []string
-	pull       bool
-	reference  string
+	SecureFlagOpts
+	reference string
+	config    string
```

Comment:
The reason for using `string` instead of slice is to be consistent with `sign` and `key` commands, https://github.com/notaryproject/notation/blob/main/cmd/notation/sign.go#L27 so that I can utilize https://github.com/notaryproject/notation/blob/main/internal/cmd/flags.go#L91 to parse the configs.